### PR TITLE
docs(p-toolbox-table): change two-way-binding hook

### PR DESCRIPTION
**Pull request by piggggggggy**
### To Reviewers
- [ ] Skipping review is not a problem (```style```, ```chore``` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [ ] Feature improvement
  - [ ] These changes require a usage change
- [x] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] Updated Storybook documents
- [x] Tested with console

### Description
- change `makeOptionalProxy` to `useProxyValue`
- tested with all `p-toolbox-table` in console

### CODE REVIEW

File: src/data-display/tables/toolbox-table/PToolboxTable.vue [hunk 1]
```diff
@@ -233,28 +233,26 @@ export default defineComponent<ToolboxTableProps>({
...
         },
         timezone: {
             type: String,
             default: 'UTC',
         },
     },
-    setup(props: ToolboxTableProps, { emit }) {
-        const vm = getCurrentInstance()?.proxy as Vue;
-
+    setup(props, { emit }: SetupContext) {
```
Comment (minseolee) [inline src/data-display/tables/toolbox-table/PToolboxTable.vue:247]: Nice Type 👍

File: src/data-display/tables/toolbox-table/PToolboxTable.vue [hunk 2]
```diff
@@ -108,11 +108,11 @@ export default defineComponent<ToolboxTableProps>({
         },
         sortBy: {
             type: String,
-            default: undefined,
+            default: '',
         },
         sortDesc: {
             type: Boolean,
-            default: undefined,
+            default: true,
```
Comment (minseolee) [inline src/data-display/tables/toolbox-table/PToolboxTable.vue:115]: Great Detail !

**Review by minseolee — APPROVED** [review]
Tested with all at console !!?!?!
great, huge, immense, enormous, tremendous.

**Review by sulmoJ — APPROVED** [review]
goooooood!!

**Pull request merged**